test/models/media_instance_test.rb

```diff
@@ -0,0 +1,15 @@
+require "test_helper"
+
+describe MediaInstance do
+  let(:media_instance) { MediaInstance.new }
+  it "must reject if there is no title" do
+    media_instance.errors.messages.include? :title
+  end
+  it "must reject if there is no media type" do
+    media_instance.errors.messages.include? :media_type
```

Comment:
Similarly, this doesn't actually check anything. When writing a test, you need to have some expectation (`must_something` or `wont_something`).

Going through the red-green-refactor cycle would have caught this. You would have tried to watch the test fail, and when it didn't, you would realize something is up.